app/src/main/res/values-v27/styles.xml

```diff
@@ -14,4 +14,5 @@
         <item name="android:windowLightStatusBar">@bool/isLightMode</item>
         <item name="android:windowLightNavigationBar">@bool/isLightMode</item>
     </style>
+
```

Comment:
No need for a new empty line ;)

Reply:
![image](https://user-images.githubusercontent.com/1634145/190004101-c9ba00f1-379f-4c5b-9b09-cd116068d675.png)